src/sentry/integrations/message_builder.py

```diff
@@ -3,8 +3,9 @@
 from abc import ABC
 from typing import Any, Callable, Mapping, Sequence
 
+from sentry.eventstore.models import Event
 from sentry.integrations.slack.message_builder import SLACK_URL_FORMAT
-from sentry.models import Event, Group, Project, Rule, Team, User
```

Comment:
I started getting weird import errors. This was imported from the wrong place anyway, so I just fixed it.